chapter_12_organising_test_files.asciidoc

```diff
@@ -547,6 +551,8 @@ Like this:
 ((("Python 3", "lambda functions")))
 `lambda` in Python is the syntax for making a one-line, throwaway function--it
 saves you from having to use `def..():` and an indented block:
+// CSANAD: the `def..` and the `():` renders apart from each other, on two
+//         lines. I couldn't find out how to keep them together.
```

Comment:
there's a special role thingie called `keep-together` - syntax is something like this,
but it might not render quite right if we use it at the same time as backticks:

```suggestion
saves you from having to use [keep-together]#`def..():`# and an indented block:.
```

could also try just `[keep-together]`def..():` hmmm

Reply:
Yeah, I tried that but didn't work. How about cheating a little with an extra word or two:
 "...from having to use **the regular** `def..():` and an indented block:" ?

Reply:
just ignore it for now, they'll pick it up in copyediting.